### ISSUE TITLE
Day night cycle? 

### ISSUE THREAD

**Issue by Pizzahaet** [ArufaTeam/Arufa#7]
Could you guys possibly add the ability to toggle Day/Night cycle?

**Comment by TheRoyalBlock** [comment]
Upvote

**Comment by Derpific** [comment]
Should be simple

**Comment by TheRoyalBlock** [comment]
Don't we already have the /time stop command?

**Comment by Pizzahaet** [comment]
It doesn't work like it should, on server reboot the time starts again or when the world is reloaded

**Comment by TheRoyalBlock** [comment]
Of course, but that's not something that we can stop...

**Comment by daniktheboss** [comment]
We can... we can make a config file and set it. Why shouldn't we be able to stop it. Smh

**Comment by TheRoyalBlock** [comment]
Right, thanks. But just make it an option in server.properties or arufa.yml instead of making yet another config file.

**Comment by xZeroMCPE** [comment]
A way to toggle \ disable day night cycle would be by implementing game rule. Which im going to be doing. Check it appropriate branch if you'd like to test for yourself.

**Comment by Pizzahaet** [comment]
Sweet man!

**Comment by daniktheboss** [comment]
Omg royalblock again!
